src/RMGTrackOutputScheme.cc

```diff
@@ -199,6 +203,11 @@ void RMGTrackOutputScheme::DefineCommands() {
       .SetParameterName("boolean", true)
       .SetDefaultValue("true")
       .SetStates(G4State_Idle);
+  fMessenger->DeclareProperty("StoreAlways", fStoreAlways)
+      .SetGuidance("Always store track data, even if event should be discarded.")
+      .SetParameterName("boolean", true)
+      .SetDefaultValue("false")
```

Comment:
we typically have the opposite value of our _true_ default as a _command_ default.

So that `/RMG/Output/Track/StoreAlways` without any parameter will turn the option on, not off.